src/test/java/org/opentripplanner/apis/transmodel/mapping/TripRequestMapperTest.java

```diff
@@ -403,6 +402,14 @@ public void testExplicitModes() {
     assertEquals(StreetMode.WALK, req.journey().transfer().mode());
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = { "transferSlack", "minimumTransferTime" })
+  public void testTransferSlack(String name) {
```

Comment:
This need a comment or a different name. I assume it is testing backward compatibility of the old name?

Reply:
It is indeed for backwards-compatibility: I clarifi€d in 468bb838f71f622367e1b3c3ae02ee199387a7fd